app/locales/en/index.json

```diff
@@ -1,163 +1,98 @@
 {
```

Comment:
I personally liked having "english.json" so if I got files back from someone they weren't all called "index.json".  Not a showstopper, but if that is possible I think it helps @alpita-masurkar manage this better.

Reply:
now named `en.json`